cli/openbb_cli/controllers/utils.py

```diff
@@ -525,26 +528,22 @@ def _get_headers(_headers: Union[List[str], pd.Index]) -> List[str]:
         if columns_to_auto_color is None and rows_to_auto_color is None:
             df = df.applymap(lambda x: return_colored_value(str(x)))
 
-    exceeds_allowed_columns = (
-        len(df.columns) > Session().settings.ALLOWED_NUMBER_OF_COLUMNS
-    )
-    exceeds_allowed_rows = len(df) > Session().settings.ALLOWED_NUMBER_OF_ROWS
+    exceeds_allowed_columns = len(df.columns) > MAX_COLS
+    exceeds_allowed_rows = len(df) > MAX_ROWS
 
     if exceeds_allowed_columns:
         original_columns = df.columns.tolist()
-        trimmed_columns = df.columns.tolist()[
-            : Session().settings.ALLOWED_NUMBER_OF_COLUMNS
-        ]
+        trimmed_columns = df.columns.tolist()[:MAX_COLS]
         df = df[trimmed_columns]
         trimmed_columns = [
             col for col in original_columns if col not in trimmed_columns
         ]
 
     if exceeds_allowed_rows:
         n_rows = len(df.index)
-        trimmed_rows = df.index.tolist()[: Session().settings.ALLOWED_NUMBER_OF_ROWS]
-        df = df.loc[trimmed_rows]
-        trimmed_rows_count = n_rows - Session().settings.ALLOWED_NUMBER_OF_ROWS
+        max_rows = MAX_ROWS
+        df = df[:max_rows]
```

Comment:
I couldn't add this on the line, but line 504 is converting the entire DataFrame to a string. It can't do that because numbers need to be numbers for the table filters to work. It would be good to have the params from `print_rich_table` exposed and accessible, those can be very useful.

Reply:
That's being done only before sending to the pywry backend, otherwise we might have not expected objects that can't be shown on the interactive tables, which breaks it.
Although that's being done, it looks like the "numerical" columns can still be correctly ordered. (or am I missing the point here?)

wdym by? `params from print_rich_table exposed and accessible` 
making those available for the user to use directly in every command? Although I agree it could be useful, wouldn't it clutter the commands a lot?